src/aind_metadata_service/server.py

```diff
@@ -61,8 +67,13 @@ async def retrieve_procedures(subject_id, pickle=False):
     merged_response = sharepoint_client.merge_responses(
         [lb_response, sp2019_response, sp2023_response]
     )
-    json_response = merged_response.map_to_json_response()
-    return json_response
+    if pickled:
+        response = pickle.dumps(merged_response)
```

Comment:
Got it. This is also a great way to separate the pickle module from the query parameter "pickle" to avoid confusion for now, but it would be nice to change the query parameter to "pickled" in a future ticket.